WordPress/gutenberg
# Site editor sidebar: add page details

## What?
Updates the existing page detail view in the site editor to include more useful information.

## Why?
https://github.com/WordPress/gutenberg/issues/50390

## Constraints

- If we want to get the author and avatar of the user who last modified the page we'll need to fetch and consult the revisions. This is possible, but needs extra work to set up the store in a follow up PR.


## Testing Instructions
1. Activate a block theme
2. Make sure you have some pages ready to role. Add featured images, leave some scheduled too.
3. Now head over to the site editor 
4. In the side bar, navigate to Design > Pages and click on the pages.
5. Check each page's details in the sidebar.
6. Overcome your bedazzlement and provide some constructive feedback. 




<img width="349" alt="Screenshot 2023-05-25 at 3 27 12 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/50d593e8-b11e-4d83-b922-67a58d292506">
<img width="346" alt="Screenshot 2023-05-25 at 3 27 00 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/573a1a31-4813-4a41-9249-afa15b4d9bd6">
<img width="348" alt="Screenshot 2023-05-25 at 3 26 32 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/371f6e29-eb29-463e-b8b1-82d468214d77">
